functions in JS/index.html

```diff
@@ -0,0 +1,18 @@
+<html>
```

Comment:
move the file to functions directory, and rename the file to functions_in_js.html.

Reply:
@t2013anurag  I guess there's no functions directory right now. Do you want me to create a functions directory and push the JS file into it?

Reply:
@shubham0109 - yes you can create one,